app.js

```diff
@@ -1,19 +1,29 @@
 const express = require("express");
 const { getTopics } = require("./controllers/topics.controllers");
 const { badPath, getEndpoints } = require("./controllers/app.controllers");
+const { getSingleArticle } = require("./controllers/articles.controllers");
 const app = express();
 app.use(express.json());
```

Comment:
Don't need this line of code quite yet so would be best practice to only have it when writing a post or patch